FEXCore/Source/Interface/IR/IR.json

```diff
@@ -1028,6 +1029,11 @@
         "NumElements": "RegisterSize / ElementSize"
       },
 
+      "FPR = LoadNamedVectorConstant u8:#RegisterSize, NamedVectorConstant:$Constant": {
+        "Desc": ["Load a named vector constant"
```

Comment:
Maybe indicate where to find the available list of constants here as well

Reply:
Added a comment where to find them.